include/crocoddyl/core/optctrl/shooting.hxx

```diff
@@ -231,6 +229,34 @@ std::vector<typename MathBaseTpl<Scalar>::VectorXs> ShootingProblemTpl<Scalar>::
   return xs;
 }
 
+template <typename Scalar>
+void ShootingProblemTpl<Scalar>::quasiStatic(const std::vector<VectorXs>& us, std::vector<VectorXs>& xs) {
```

Comment:
`us` should not be constant, and `xs` should not be constant.

Reply:
done